Makefile

```diff
@@ -144,10 +153,13 @@ uninstall: manifests kustomize ## Uninstall CRDs from the K8s cluster specified
 deploy: manifests kustomize ## Deploy controller to the K8s cluster specified in ~/.kube/config.
 	cd config/manager && $(KUSTOMIZE) edit set image controller=${IMG}
 	$(KUSTOMIZE) build config/default | $(KUBECTL) apply -f -
+	$(KUSTOMIZE) build config/csi-rbac | $(KUBECTL) apply -f -
```

Comment:
You don't need this to do this.
You can, instead, add the new csi folder into default/kusomizatoin.yaml as all other folders.